skills/dynamo-router-starter/references/router-modes.md

```
@@ -0,0 +1,58 @@
+Kubernetes frontend env equivalents:
+
+| Purpose | Env |
+| --- | --- |
+| Enable KV router | `DYN_ROUTER_MODE=kv` |
+| Disable worker KV event consumption for approximate mode | `DYN_ROUTER_USE_KV_EVENTS=false` |
+| Enable load-aware behavior | `DYN_ROUTER_LOAD_AWARE=true` |
+| Set router randomness | `DYN_ROUTER_TEMPERATURE=<float>` |
+| Set KV cache block size | `DYN_KV_CACHE_BLOCK_SIZE=<size>` |
+| Tune KV overlap credit | `DYN_ROUTER_KV_OVERLAP_SCORE_CREDIT=<float>` |
+| Scale prefill load | `DYN_ROUTER_PREFILL_LOAD_SCALE=<float>` |
+| Set queue policy | `DYN_ROUTER_QUEUE_POLICY=fcfs|wspt|lcfs` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)